src/Promotions/InPluginUpsells/AddonsAdminPage.php

```diff
@@ -29,34 +31,29 @@ public function register()
      */
     public function loadScripts()
     {
-        wp_enqueue_script(
-            'give-in-plugin-upsells-addons',
-            GIVE_PLUGIN_URL . 'assets/dist/js/admin-upsell-addons-page.js',
-            ['wp-element', 'wp-i18n', 'wp-hooks'],
-            GIVE_VERSION,
-            true
-        );
+        EnqueueScript::make('give-in-plugin-upsells-addons', 'assets/dist/js/admin-upsell-addons-page.js')
+            ->loadInFooter()
+            ->registerTranslations()
+            ->registerLocalizeData(
+                'GiveAddons',
+                array_merge(
```

Comment:
It might be more readable to set this nested add-ons data as a variable, which is then passed to the `registerLocalizeData()` method.

Reply:
fixed: https://github.com/impress-org/givewp/compare/4ee3cdc25bd4...21f729ed83ab